docs/ops/disting.toml

```diff
@@ -209,7 +209,7 @@ short = "send MIDI Note On message on channel `x` for note `y` (0..127) and velo
 prototype = "EX.M.NO x"
 short = "send MIDI Note off message for note `x` (0..127)"
 
-["EX.M.NO"]
```

Comment:
ah, bad copy/paste on my part, thanks for catching it! could you also add `#` in the prototype below?

Reply:
i see other places i missed, i’ll submit a PR to fix it!